homeassistant/components/device_tracker/meraki.py

```diff
@@ -94,7 +94,10 @@ def post(self, request):
     def _handle(self, hass, data):
         for i in data["data"]["observations"]:
             data["data"]["secret"] = "hidden"
-            mac = i["clientMac"]
+            if data['type'] == 'BluetoothDevicesSeen':
+                mac = 'BLE_' + i["clientMac"]
```

Comment:
Why do you want to change the address?

Reply:
Personally its nice to be able to seperate network clients and BLE clients.
Since the bluetooth trackers in HA already uses theese prefixes to seperate them from network appliances i thought it was a good idea.

Reply:
I think that is just an implementation detail of those trackers, and probably shouldn't be exposed to users. I don't think the BLE tracker even needs to use this prefix, so it should be reworked.

I'll leave this for a third opinion though.

Reply:
I am not sure i am missunderstanding you but in **known_devices.yaml** 
This is automatically created by the bluetooth tracker component.

myphone:
  hide_if_away: true
  icon: mdi:bluetooth
  mac: BT_6C:72:E7:2E:D8:9C
  name: "Myphone"
  picture:
  track: true
  vendor: Apple, Inc.

So if the meraki all of a sudden starts entering the bluetooth mac adresses in **known_devices.yaml** without a prefix it will get confusing.

But please explain if im understanding wrong... im new here ;-)

Reply:
The prefix is something these home assistant platforms add to the mac address. The original mac address does not contain that prefix. The prefix is used in the platforms to keep track of Bluetooth devices. It's not meant for users.

Reply:
The BT prefix in bluetooth tracker is not there for internal use. Its there to be represented in known_devices.yaml.  A macaddress is **unique** so saving with a prefix in known_devices can for what i see only be for **users** since the platform has the unique mac and without prefix would still find it in known_hosts.

Reply:
No. The prefix is used to internally keep track of bluetooth devices, also after restart and loading from known_devices.yaml. This is to be able to only query certain devices for update:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/device_tracker/bluetooth_tracker.py#L80-L82

Reply:
So Meraki BLE tracker should not be able to work side by side with the Blutotooth LE tracker? Since they obviously then will create two seperate entities...

Reply:
Since the Bluetooth LE tracker doesn't seem to have a valid use for the prefix, it makes more sense to remove it from that tracker. But I think someone more involved in these platforms should comment.